source/docs/03-metadata.xml

```diff
@@ -1444,6 +1444,82 @@
                      <p>Watermark</p>
                      <p>Stamp</p>
                      <p>physMedium</p>-->
+                     <p>Encoding the exent and the dimensions of a source:</p>
+                     <p><specList>
+                        <specDesc key="extent"/>
+                     </specList></p>
+                     <p>The element <gi scheme="MEI">extent</gi> is used to express size in terms other than physical dimensions. This can be done in units such as ‘number of pages’, ‘number of folios’, ‘records’, ‘bytes’, ‘physical components’, etc. For specifying the physical size of the material, for example, height and width, the use of the <gi scheme="MEI">dimensions</gi> element is recommended.</p>
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample085.txt" parse="text"/></egXML>
+                        </figure>
+                     </p>
+                     <p><specList>
+                        <specDesc key="dimensions"/>
+                     </specList></p>
+                     <p>The <gi scheme="MEI">dimensions</gi> contain information about the physical size of a source or entity in general. Usually the dimensions are represented by numerical data. The elements <gi scheme="MEI">height</gi>, <gi scheme="MEI">width</gi>, <gi scheme="MEI">depth</gi>, and <gi scheme="MEI">dim</gi> are available for circumstances that require the capture of the individual dimensions of an object. Do not confuse this element with the <gi scheme="MEI">extent</gi> element, which is used to indicate the quantity of described materials.</p>
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample086.txt" parse="text"/></egXML>
+                        </figure>
+                     </p>
+                     <p><specList>
+                        <specDesc key="dim"/>
+                     </specList></p>
+                     <p>Another way of encoding is to use the element <gi scheme="MEI">dim</gi>, that can be used to represent any single dimensional specification. The attribute <att>form</att> is required. When appropriate the elements <gi scheme="MEI">height</gi>, <gi scheme="MEI">width</gi>, and <gi scheme="MEI">depth</gi> are preferred to specify the content of <gi scheme="MEI">dim</gi>.</p>
```

Comment:
```suggestion
                     <p>Another way of encoding is to use the element <gi scheme="MEI">dim</gi>, which can be used to represent any dimensional specification in a generic way. The attribute <att>form</att> is required. With regard to structured metadata, the use of the <gi scheme="MEI">height</gi>, <gi scheme="MEI">width</gi>, and <gi scheme="MEI">depth</gi> elements as the content of <gi scheme="MEI">dim</gi> is advisable.</p>
```

When I read through the guidelines sometimes it is not clear to me which goal is pursued with the guidelines. Is it just the objective explanation of the elements (A can do this and B can do that) or is it also to convey more content (the difference between A and B is...)? Furthermore, I often ask myself whether it is appropriate to show directions (such as "Practice Recommendations")

Reply:
I think all are fine, but there needs to be a clear indication of whose practice you’re recommending, and why, if you are going to do that. In most cases it is probably clearer to indicate the benefits and compromises, rather than a specific recommendation. Or if something is there for a very specific reason and should not otherwise be used even if it seems, at first sight, to be useful. (MEI is full of these things, where things are allowed inside other things only because they meet a very specific need…)

Reply:
Thank you for this insight. This helps me a lot!